src/Apps/CollectorProfile/Routes/Saves2/Components/SelectArtworkListsModal/SelectArtworkListsHeader.tsx

```diff
@@ -38,23 +38,22 @@ export const SelectArtworkListsHeader: FC = () => {
         </Button>
       </Flex>
 
-      <Spacer y={2} />
-
       {state.recentlyAddedList && (
-        <Message
-          variant="success"
-          title={t(
-            "collectorSaves.selectArtworkLists.header.recentlyAddedList.title"
-          )}
-          mx={-2}
```

Comment:
I would expect a negative margin to have a comment explaining why it exists, since it's by default working around something that probably exists for a good reason.

Reply:
Could this be to open modal full-screen?

| Before | Your changes |
|---|---|
| <img width="400" alt="Screenshot 2023-06-22 at 22 05 33" src="https://github.com/artsy/force/assets/3934579/f9aaf368-2484-4a68-8929-05f4bccea892"> | <img width="406" alt="Screenshot 2023-06-22 at 22 05 44" src="https://github.com/artsy/force/assets/3934579/fd7f460a-ac8d-4e8c-885a-8ede3d3a65d3"> |

Reply:
This is a different margin, actually. This one is for `Message`.

Reply:
Though, it seems to me that external dialog margins are not something that should be customized. If there's a problem we should change it in Palette.